safe_cli/operators/safe_operator.py

```diff
@@ -325,6 +335,34 @@ def load_cli_owners(self, keys: List[str]):
             except ValueError:
                 print_formatted_text(HTML(f"<ansired>Cannot load key={key}</ansired>"))
 
+    def load_ledger_cli_owners(self, legacy_account: bool = False):
```

Comment:
Return type

Reply:
Maybe for next iteration we should allow people to use a custom derivation path like:

`> load_ledger_cli_owners DERIVATION_PATH`